releases/release-1.17/README.md

```diff
@@ -0,0 +1,196 @@
+by Code Freeze. The exception approval is the responsibility of the SIG or SIGs
+labeled in the pull request. The release team may intervene or deny the request
+only if it poses a risk to release quality, or could negatively impact the overall
+timeline. Changes introduced at this point should be well-tested,
+well-understood, limited in architectural scope, and low risk.  All of those
+factors should be considered in the approval process.  Enhancements on an
+feature branch with documentation, test cases, and passing CI are more likely to
+be accepted.
+
+### Pruning
+
+Enhancements that are partially implemented and/or lack sufficient tests may be
+considered for pruning beginning after [Code Freeze], unless they've been
+granted an [Exception].
+
+The release team will work with SIGs and enhancements owners to evaluate each
+case, but for example, pruning could include actions such as:
+
+* Disabling the use of a new API or field
+* Switching the default value of a flag or field
+* Moving a new API or field behind an Alpha Enhancements gate
+* Reverting commits or deleting code
+
+This should occur before 1.17.0-beta.1 is cut so we have time to gather signal
+on whether the system is stable in this state. These are considered drastic
+measures, so the release team will strive to coordinate at-risk work with SIGs
+before this time. The goal is to make code freeze, and overall project
+transparency, enforceable despite the lack of a consistently used feature branch
+process.
+
+### Docs
+
+If an enhancement needs documentation, enter "Yes" in the enhancement tracking
+spreadsheet and add a link to the documentation PR. You can open documentation
+PRs in the [kubernetes/website] repository. If you have questions, the release
+documentation lead, or representatives from SIG-Docs will be happy to assist
+you.
+
+For documentation PRs:
+
+* Open PRs against the dev-1.17 branch based off of the 1.17 release PR. The
+  documentation workflow uses feature branches for release documentation, rather
+than basing from master. **Be sure to open your PR against the release branch**.
+* Add your PR to the 1.17 Release milestone.
+
+[Enhancements Freeze]: #enhancements-freeze
+[Burndown]: #burndown
+[Code Freeze]: #code-freeze
+[Code Thaw]: #code-thaw
+[Exception]: #exceptions
+
+[kubernetes-release-team@]: https://groups.google.com/forum/#!forum/kubernetes-release-team
+[kubernetes-sig-release@]: https://groups.google.com/forum/#!forum/kubernetes-sig-release
+[#sig-release]: https://kubernetes.slack.com/messages/sig-release/
+[kubernetes-release-calendar]: https://bit.ly/k8s-release-cal
+[kubernetes/kubernetes]: https://github.com/kubernetes/kubernetes
+[kubernetes/enhancements]: https://github.com/kubernetes/enhancements
+[kubernetes/website]: https://github.com/kubernetes/website
+
+[master-blocking]: https://testgrid.k8s.io/sig-release-master-blocking#Summary
+[master-upgrade]: https://testgrid.k8s.io/sig-release-master-upgrade#Summary
```

Comment:
Replace this one with:
[master-informing]: https://testgrid.k8s.io/sig-release-master-informing